### DISCUSSION TITLE
Is there a way to auto mount when Mountain Duck starts?

### DISCUSSION THREAD

**Discussion by Andesgong** [iterate-ch/cyberduck#15424] [Q&A]
In CloudMounter, there's an option to auto connect when the app starts up(not when the computer starts), when saves me a bunch   of time. I'm wondering is there a way in Mountain Duck that does the same thing?

**Comment by dkocher** [comment]
Choose _Mountain Duck → Preferences… → General → Save Workspace_. Make sure to quit the application without ejecting or disconnecting the volumes to make this work.

**Comment by diniramdhni** [comment]
You may achieve a similar outcome by using your operating system's startup or login items features. Mountain Duck may be added to the login items list in the "Users & Groups" settings on macOS, and a shortcut in the Startup folder on Windows. Mountain Duck will then run immediately when you log in, and if it is set to remember mounted disks, it will attempt to reconnect them.
